packages/react-instantsearch-hooks-web/src/widgets/Stats.tsx

```diff
@@ -8,6 +8,7 @@ import type {
   StatsTranslationOptions,
 } from '../ui/Stats';
 import type { UseStatsProps } from 'react-instantsearch-hooks';
+import { warn } from '../../../react-instantsearch-hooks/src/lib/warn';
```

Comment:
```suggestion
import { warn } from 'react-instantsearch-hooks/src/lib/warn';
```

this needs to be exported to be usable I believe (otherwise use `/es/` path but that's fragile)